src/framework/theme/components/select/_select.component.theme.scss

```diff
@@ -57,6 +57,11 @@
   nb-option {
     user-select: none;
 
+    &.active {
```

Comment:
`autocomplete-*` theme properties shouldn't be used outside of autocomplete theme file

Reply:
Fixed